cla_backend/settings/base.py

```diff
@@ -307,12 +307,15 @@ def traces_sampler(sampling_context):
 LOGIN_FAILURE_LIMIT = 5
 LOGIN_FAILURE_COOLOFF_TIME = 60  # in minutes
 
-# Django rest-framework-auth
+# Django rest-framework-overrides
```

Comment:
What does this mean?

Reply:
This section of the config previously only contained rest framework auth config overrides, it now contains general overrides